File: Python3/maior_menor.py
"""Problema: Fazer um algoritmo que analize 3 valores inteiros (através das variáveis n1, n2 e n3),
e informe qual o maior e qual o menor deles. """

n1 = int(input("Digite o valor para N1: "))
n2 = int(input("Digite o valor para N2: "))
n3 = int(input("Digite o valor para N3: "))
if n1 >= n2 and n1 >= n3:
   print("O maior é :",n1)
elif n2 >= n1 and n2 >= n3:
   print("O maior é :",n2)
else:
   print("O maior é :",n3)
if n1 <= n2 and n1 <= n3:
   print("O menor é: ",n1)
elif n2 <= n1 and n2 <= n3:
   print("O menor é :",n2)
else:
   print("O menor é :",n3)

File: java/maior_menor.java
import java.util.Scanner;

class maior_menor {
   public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);

      System.out.println("Digite o valor para N1: ");
      int n1 = sc.nextInt();
      System.out.println("Digite o valor para N2: ");
      int n2 = sc.nextInt();
      System.out.println("Digite o valor para N3: ");
      int n3 = sc.nextInt();

      if ((n1 >= n2) && (n1 >= n3))
         System.out.println("o maior e o : " + n1);
      else if ((n2 >= n1) && (n2 >= n3))
         System.out.println("o maior e o : " + n2);
      else
         System.out.println("o maior e o : " + n3);

      if ((n1 <= n2) && (n1 <= n3))
         System.out.println("o menor e o : " + n1);
      else if ((n2 <= n1) && (n2 <= n3))
         System.out.println("\no menor e o : " + n2);
      else
         System.out.println("\no menor e o : " + n3);

      sc.close();

   }
}
